content/docs/git/ide.md

```diff
@@ -0,0 +1,248 @@
+| Code | \`code`
+| Link | \[title](https://www.example.com)
+| Image | \![alt text]\(image.jpg)
+
+[Additional Markdown commands](https://www.markdownguide.org/cheat-sheet/)
+
+
+### 1. Fork and clone your new Journal
+
+First, navigate to the directory where you want to clone the journal. (If you don't already have a designated folder for class materials, create a directory for this and navigate into it.) Cloning a repository means creating a local copy of another repository, usually one that is stored on a remote server like GitHub.
+
+Fork [this repository](https://github.com/ReDI-School/autumn-2021-intro-to-programming-md-journal). In the top right corner, click `Fork`. You may be prompted with a question like "Where should we fork this repository to?" If so, click your GitHub username; this will create a copy of the repository to your account.
+
+You should then be automatically taken to the main page of this repository. 
+
+On the upper right side, click `Code` (in green) and copy the `HTTPS` link. Then in your terminal run `git clone <link-you-copied>`. For example, this should look something like the following (with `github-username` replaced with your own github username):
+
+```sh
+git clone https://github.com/ReDI-School/autumn-2021-intro-to-programming-md-journal
+```
+
+Check that this directory now exists. _(Hint: list the contents of the directory you are in, and make sure you see the cloned folder.)_
+
+Move into that directory (small hint -> write `cd autumn` and then press the key `TAB` and it will autocomplete):
+
+```sh
+cd autumn-2021-intro-to-programming-md-journal
+```
+
+Check the status of the git repository:
+
+```sh
+git status
+```
+
+You should see output something like:
+
+```sh
+On branch main
+Your branch is up to date with 'origin/main'.
+
+nothing to commit, working tree clean
+```
+
+If so, great! You have successfully forked and cloned this repository. If not, check in with someone in your group or raise your hand so one of the teachers to help before moving on to the next steps.
+
+### 2. Checkout a new branch
+
+One of the great things about Git is being able to work on the same repository as others at the same time. A typical way of doing this when working on a project is by creating and checking out a new branch (via `git checkout -b <branch-name>`), making changes and committing your work, and pushing this work to your separate branch. Once you are ready to publish your changes to the `main` branch, you can create a pull request (PR) to have others review your changes and merge them to the `main` branch.
+
+Even though you are working on your own repository alone (since you forked, this is your own repository), let's practice creating a new branch so we can make some changes there and eventually merge this back to the `main` branch. Let's call this branch `<your-github-username>/test-branch` (leave out the <>, for example, for me, I would call the branch `omarsotillofranco/journal1`):
```

Comment:
maybe remove omar's username, something like: "(leave out the <>, for example, if your github username is `alice500`, you would call the branch `alice500/journal1`):"

Reply:
It slipped through 😄